### PR TITLE
Support UUID in CUDA_VISIBLE_DEVICES

### PR DESCRIPTION
**Pull request by pentschev**
Originally seen first in https://github.com/rapidsai/dask-cuda/issues/435 , this should allow specifying GPUs by their UUIDs in `CUDA_VISIBLE_DEVICES`.

### CODE REVIEW

**Comment by quasiben** [comment]
rerun tests

File: dask_cuda/utils.py [hunk 1]
```diff
@@ -388,3 +388,57 @@ def f(x):
 
 def all_to_all(client):
     return client.sync(_all_to_all, client=client, asynchronous=client.asynchronous)
+
+
+def parse_cuda_visible_device(dev):
+    """Parses a single CUDA device identifier
+
+    A device identifier must either be an intenger, a string containing an
```
Comment (madsbk) [inline dask_cuda/utils.py:396]:
```suggestion
    A device identifier must either be an integer, a string containing an
```

**Review by madsbk — APPROVED** [review]
Other than a typo, it looks good to me

**Comment by pentschev** [comment]
Thanks @madsbk !

**Pull request merged**